department-of-veterans-affairs/abd-vro
# EP Merge App: create helm chart for deployments

## What was the problem?

The Employee Experience Team's newest project, EP Merge App, was missing a helm chart which is required for deployment [per LHDI standards](https://hub.lighthouse.va.gov/docs/default/component/delivery-platform-docs/Lighthouse-delivery-infrastructure-tools/Lighthouse-DI-tools/).

Associated tickets or Slack threads:
- #2403 

## How does this fix it?[^1]
This PR copies the existing max CFI helm chart used by the EE Team, essentially changing only the port numbers/name/description.

## How to test this PR
- GitHub Actions update-deployment workflow
- Run `helm/deploy.sh` against a local kubernetes cluster


[^1]: [Pull-Requests guidelines](https://github.com/department-of-veterans-affairs/abd-vro/wiki/Pull-Requests). If PR is significant, update [Current Software State](https://github.com/department-of-veterans-affairs/abd-vro/wiki/Current-Software-State) wiki page.
[^secrel]: To check if a PR will succeed in the SecRel workflow, [test PRs in the SecRel pipeline](https://github.com/department-of-veterans-affairs/abd-vro-internal/wiki/Secure-Release-process#to-test-prs-in-the-secrel-pipeline).
